doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -208,12 +209,14 @@ Bluetooth mesh samples
 
   * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340`.
   * Updated some samples to use DTS overlays instead of KConfig configuration files for setting up external flash memory.
+  * When publishing is enabled for a model a corresponding status message shall be published immediately after the state transition ends. This had to be done by the application before.
+  * Aligned the Silvair EnOcean Proxy Server model implementation with rev 1.2 of the Silvair EnOcean Switch Mesh Proxy Server specification.
+  * Fixed an issue where Sensor Client API can be used as non-blocking by passing NULL to the arguments that are used to fill the response. 
```

Comment:
Why all these 3 points under `Bluetooth mesh samples` section? Should they be under `Bluetooth mesh` section like it was before?

Reply:
They are moved to a section under Libraries.